content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/expose-services.mdx

```diff
@@ -0,0 +1,118 @@
+---
+docType: "Chapter"
+chapterTitle: "Ingressing and Egressing with Linkerd"
```

Comment:
@Nikhil-Ladha I am not sure, should I change the `chapterTitle` to `Exposing services through Linkerd Ingress` or something else and move `Ingressing and Egressing with Linkerd` down to an `h2` heading below the beginning of `ChapterStyle` tag.
What do u suggest?

Reply:
Yeah, that's a great suggestion. Please do so.

Reply:
cool